File: leetcode/1227_airplane_seat_assignment_probability.cpp
// Copyright (c) 2019 Jacopo Notarstefano

#include <cassert>

using namespace std;

class Solution {
 public:
    double nthPersonGetsNthSeat(int n) {
        return (n == 1) ? 1 : 0.5;
    }
};

int main() {
    auto solution = Solution();

    assert(1 == solution.nthPersonGetsNthSeat(1));
    assert(0.5 == solution.nthPersonGetsNthSeat(2));
    assert(0.5 == solution.nthPersonGetsNthSeat(3));
}

File: leetcode/1227_airplane_seat_assignment_probability.py
# -*- coding: utf-8 -*-


class Solution(object):
    def nthPersonGetsNthSeat(self, n):
        return 1 if n == 1 else 0.5


if __name__ == '__main__':
    solution = Solution()

    assert 1 == solution.nthPersonGetsNthSeat(1)
    assert 0.5 == solution.nthPersonGetsNthSeat(2)
    assert 0.5 == solution.nthPersonGetsNthSeat(3)
